### PR TITLE
LUD-21: LNURL-verify

### PR DESCRIPTION
**Pull request by reneaaron**
This PR adds LNURL-verify specs.

**Discussion**
#182 

I couldn't find any info about how to obtain a LUD number, so I just took the next free one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ These are all the individual documents describing each small piece of protocol t
 | [18][18] | Payer identity in `payRequest` protocol.                    | [Alby][alby], [BitBanana][bitbanana], [Blixt][blixt], [cliché][cliche], [OBW][obw], [ZBD Discord][zbd], [ZBD Telegram][zbd] |
 | [19][19] | Pay link discoverable from withdraw link.                   | [Blixt][blixt], [CoinCorner][coincorner], [OBW][obw], [LNbits][lnbits] |
 | [20][20] | Long payment description for pay protocol.                  | [Alby][alby], [BitBanana][bitbanana], [Blixt][blixt], [Clams][clams], [cliché][cliche], [Mash][mash], [OneKey][onekey], [Phoenix][phoenix] |
+| [21][21] | `verify` invoice payments                                   | [Alby][alby], [Mutiny][mutiny], [Stacker.News][stacker.news] |
```
Comment (AnthonyRonning) [inline README.md]:
```suggestion
| [21][21] | `verify` invoice payments                                   | [Alby][alby], [Mutiny][mutiny], [Stacker.News][stacker.news], [Zaprite][zaprite] |
```

Zaprite also supports this already too. Would need to add them to the links below too. 

```
[zaprite]: https://zaprite.com
```

File: 21.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## LNURL-verify
+
+An optional field `verify` is added to the `callback` response:
+
+```diff
+{
+  "status": "OK",
+  "routes": [],
+  "pr":     "lnbc10...",
++ "verify": "https://example.com/verify/894e7f7e...",
```
Comment (AnthonyRonning) [inline 21.md]:
```suggestion
  "verify": "https://example.com/verify/894e7f7e...",
```

Extra `+` in there accidentally?

**Review by AnthonyRonning — COMMENTED** [review]
Some minor nits

File: 21.md (hunk 2 above)
Comment (reneaaron) [reply 21.md]: This is rendering the diff and showing which line was added. I guess it made more sense in the issue to describe the proposed changes, will remove it :ok_hand: 

image

**Review by AnthonyRonning — APPROVED** [review]

**Review by hsjoberg — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -25,6 +25,7 @@ These are all the individual documents describing each small piece of protocol t
 | [18][18] | Payer identity in `payRequest` protocol.                    | [Alby][alby], [BitBanana][bitbanana], [Blixt][blixt], [cliché][cliche], [OBW][obw], [ZBD Discord][zbd], [ZBD Telegram][zbd] |
 | [19][19] | Pay link discoverable from withdraw link.                   | [Blixt][blixt], [CoinCorner][coincorner], [OBW][obw], [LNbits][lnbits] |
 | [20][20] | Long payment description for pay protocol.                  | [Alby][alby], [BitBanana][bitbanana], [Blixt][blixt], [Clams][clams], [cliché][cliche], [Mash][mash], [OneKey][onekey], [Phoenix][phoenix] |
+| [21][21] | `verify` invoice payments                                   | [Alby][alby], [Mutiny][mutiny], [Stacker.News][stacker.news], [Zaprite][zaprite] |
```
Comment (hsjoberg) [inline README.md]: nit: Perhaps make it clear that this is related to LNURL-pay.

```suggestion
| [21][21] | `verify` LNURL-pay payments                                 | [Alby][alby], [Mutiny][mutiny], [Stacker.News][stacker.news], [Zaprite][zaprite] |
```

**Pull request merged**